tokio/src/process/mod.rs

```diff
@@ -1211,31 +1212,40 @@ impl Child {
     }
 }
 
-/// The standard input stream for spawned children.
-///
-/// This type implements the `AsyncWrite` trait to pass data to the stdin handle of
-/// handle of a child process asynchronously.
-#[derive(Debug)]
-pub struct ChildStdin {
-    inner: imp::ChildStdio,
+pin_project! {
+    /// The standard input stream for spawned children.
+    ///
+    /// This type implements the `AsyncWrite` trait to pass data to the stdin handle of
+    /// handle of a child process asynchronously.
+    #[derive(Debug)]
+    pub struct ChildStdin {
+        #[pin]
```

Comment:
There are still a bunch of unnecessary uses of `pin_project!`.

Reply:
Ah, thanks, fixed!